Corona-Warn-App/src/main/res/layout/home_card_container_layout.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/card_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:backgroundTint="@color/colorSurface1"
+    android:layout_marginStart="@dimen/spacing_small"
+    android:layout_marginEnd="@dimen/spacing_small"
+    android:layout_marginTop="@dimen/spacing_tiny"
+    android:layout_marginBottom="@dimen/spacing_tiny"
+    android:foreground="?selectableItemBackground"
+    android:focusable="true"
+    app:cardElevation="@dimen/elevation_strong"
```

Comment:
I think I introduced an inconsistency with the intended elevation here, as I spotted some code that explicitly set the risk card to `10dp` (e.g. `TracingDisabledBox`). I added this line because the code didn't work anymore (without me knowing that it existed and wondering a little), because it would have to set `cardElevation` for the `CardView` instead of `elevation`. Maybe you could try to find out what the intended elevation is, and make sure to set it correctly.